ecommerce/migrations/0031_discount_payment_type.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.15 on 2023-01-25 11:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ecommerce', '0030_alter_discount_for_flexible_pricing'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='discount',
+            name='payment_type',
+            field=models.CharField(choices=[('B2C', 'Marketing'), ('B2B', 'Sales'), ('FA', 'Financial Assistance'), ('CS', 'Customer Support')], max_length=3, null=True),
```

Comment:
I need to confirm this list. Please don't merge until I've signed off.

Reply:
@pdpinch Do you have an update on this?

Reply:
Need to add one more: "staff"